deployability/modules/testing/tests/test_agent/test_basic_info.py

```diff
@@ -6,8 +6,8 @@
 import re
 
 from ..helpers.agent import WazuhAgent, WazuhAPI
-from ..helpers.constants import WAZUH_ROOT
-from ..helpers.generic import HostConfiguration, HostInformation, GeneralComponentActions, Waits
+from ..helpers.constants import WAZUH_ROOT, WINDOWS_ROOT_DIR, MACOS_ROOT_DIR
+from ..helpers.generic import HostInformation, GeneralComponentActions, Waits
 from modules.testing.utils import logger
```

Comment:
Place this import before `..helpers` import statements.

Reply:
Done in  https://github.com/wazuh/wazuh-qa/pull/5310/commits/65ca9b4687cd5d98cb49b609aa5d1f9a662d666c